api/arceos_posix_api/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+#![cfg_attr(all(not(test), not(doc)), no_std)]
+#![feature(ip_in_core)]
+#![feature(int_roundings)]
+#![feature(result_option_inspect)]
+#![feature(doc_cfg)]
+#![feature(doc_auto_cfg)]
+#![allow(clippy::missing_safety_doc)]
+#![allow(missing_docs)]
+#![allow(non_camel_case_types, non_upper_case_globals)]
+
+extern crate axruntime;
+
+#[macro_use]
+extern crate axlog;
+
+#[cfg(feature = "alloc")]
+extern crate alloc;
+
+#[macro_use]
+mod utils;
+
+mod config;
+mod imp;
+
+/// cbindgen:ignore
+#[rustfmt::skip]
+#[path = "./ctypes_gen.rs"]
+#[allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals, clippy::upper_case_acronyms)]
+pub mod ctypes;
```

Comment:
Add `missing_docs` in `#[allow()]` and we can remove the attribute at the file top.